### DISCUSSION TITLE
Unable To Access GitHub Pages

### DISCUSSION THREAD

**Discussion by kjrm209** [community/community#65670] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, and good day to you !

I seek help with accessing my personally owned website on GitHub Pages.

Upon checks of the my website's repository on github.com which consists of only images, I found that the uploads of those images was unsuccessful, due to the processing error displayed here: 

![SomeChecksWereNotSuccessful](https://github.com/community/community/assets/61448722/42e826af-787e-4349-8e5a-8bcf03fee959)

When I click on the link "details" for the statement "page build and deployment / build (dynamic)", the set of syntax is displayed under the "Jobs" information pane here: 

![BuildWithJekyllConversionError](https://github.com/community/community/assets/61448722/7dd9f999-9a8e-4c63-bff2-12046ca5675e)

I am unable to see the URL of my website which should usually appear on the dashboard here: 

![GitHubPagesMissingRepositoryURL](https://github.com/community/community/assets/61448722/c2da6eff-fbfd-45ef-b910-45cf157e3190)

When I do write the full length of the said website in the URL box which leads to a specific image that has not been inserted to an index.html file, I encounter this Error 404 message:

![Error404- jpgFile](https://github.com/community/community/assets/61448722/31ec0b17-ae28-49fe-9a3c-db67741ed562)

I would appreciate any assistance which you are able to provide. If there is anything I can offer in return for your good help, please let me know too.

Thank you.

Best regards,
Kieran

**Comment by kjrm209** [comment]
This is what the my repository file list looks like:

![RepositoryFileListView](https://github.com/community/community/assets/61448722/8ff2566a-4c56-45c2-9ee4-ef5f7d12b2a0)

**Comment by jesse-sealand** [comment]
You appear to have two issues going on.

1. The most important one is that the repo name must be "kjrm209.github.io" for your page. 

2. Your jekyll site is reporting missing file and folder errors.

I would first start with changing the repo name to the correct format, then work on your jekyll site next.
